### DISCUSSION TITLE
Gff3 file cannot be properly displayed

### DISCUSSION THREAD

**Discussion by Abieskawa** [GMOD/jbrowse-components#3787] [Q&A] [answered]
I am a new user of Jbrowse 2.6.1. When I try to import gff3 file from [https://phytozome-next.jgi.doe.gov/info/Sbicolor_v3_1_1](url) and its genome, i fail to display it on my MacBook m2 chip laptop. I have already index the file with following code:
`sort -k1,1 -k4,4n Sorghum_bicolor.Sorghum_bicolor_NCBIv3.56.gff3 > sorted.gff3`
`bgzip sorted.gff3`
`tabix -p gff sorted.gff3.gz`
<img width="1428" alt="Screenshot 2023-07-07 at 1 36 16 PM" src="https://github.com/GMOD/jbrowse-components/assets/47135861/b65e4e32-ef83-45b4-8630-649d7ba078a8">

**Comment by cmdcolin** [comment, marked as answer]
The most common reason for data "not showing up" is that the reference names from your gff don't match the names in the FASTA. It is also very strict being case sensitive and such as "chr1" does not match "Chr1" or "Chr01" or "1". I can see you mention in your post using "Sorghum_bicolor.Sorghum_bicolor_NCBIv3.56.gff3" which appears to be from Ensembl. This file uses just "1".

You can either hand edit the GFF3 to change "1"->"Chr01" or create a "ref name aliases" configuration for your jbrowse. See https://jbrowse.org/jb2/docs/config_guides/assemblies/#configuring-reference-name-aliasing
